fitbenchmarking/controllers/base_controller.py

```diff
@@ -283,7 +283,7 @@ def cleanup(self):
 
         The flag corresponds to the following messages:
 
-        .. automethod:: fitbenchmarking.controllers.base_controller.Controller.flag()
-                      :noindex:
+        .. automethod:: fitbenchmarking.controllers.base_controller.Controller.flag()  # noqa: E501
+                :noindex:
         """
```

Comment:
I think the triple quote is all treated as a single line, so the noqa here is part of the string not a comment.
```suggestion
        .. automethod:: fitbenchmarking.controllers.base_controller.Controller.flag()  # noqa: E501
                :noindex:
        """  # noqa: E501
```